src/main/resources/plugin.yml

```diff
@@ -2,9 +2,9 @@ name: ${project.name}
 version: ${project.version}
 main: org.bitbucket.ucchy.undine.UndineMailer
 description: UndineMailer - Mail Plugin
-softdepend: [Vault,PermissionsEx]
+softdepend: [Vault,LuckPerms]
 author: ucchy
-api-version: 1.13
+api-version: 1.17
```

Comment:
I suppose this is necessary to support sending newer blocks? But otherwise I don't see why an api-version bump is needed.

Reply:
In my opinion, we should take what we can get when we can get it.
The reason is that I don't see the benefit of using an older API version.

Reply:
This will prevent the plugin from working on these older versions though. Hence why I'd leave this to the maintainer to decide if compatibility should be broken or not.

Reply:
80460f910cf2eb4d412ab5fb6aca681184d70458 Fixed,
Thank you for reviewing